extensions/deviceicon/battery.py

```diff
@@ -43,13 +44,9 @@
 
 _UP_TYPE_BATTERY = 2
 
-_UP_STATE_UNKNOWN = 0
-_UP_STATE_CHARGING = 1
-_UP_STATE_DISCHARGING = 2
-_UP_STATE_EMPTY = 3
-_UP_STATE_FULL = 4
-_UP_STATE_CHARGE_PENDING = 5
-_UP_STATE_DISCHARGE_PENDING = 6
+_UP_STATE_UNKNOWN = UPowerGlib.DeviceState.UNKNOWN
+_UP_STATE_CHARGING = UPowerGlib.DeviceState.CHARGING
+_UP_STATE_DISCHARGING = UPowerGlib.DeviceState.DISCHARGING
```

Comment:
Couldn't we just use the UPowerGlib constants directly?  Find and replace is good :)